packages/db-migrations/databases/fxa/patches/patch-142-143.sql

```diff
@@ -0,0 +1,87 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('142');
+
+-- Adds the providerId which maps to the third party auth provider used to generate session
+ALTER TABLE `sessionTokens`
+ADD COLUMN `providerId` TINYINT DEFAULT NULL,
+ALGORITHM = INPLACE, LOCK = NONE;
```

Comment:
Could try INSTANT instead of INPLACE.

Reply:
Good call, read the docs and looks like `INSTANT` can [used](https://dev.mysql.com/blog-archive/mysql-8-0-innodb-now-supports-instant-add-column/).